### PR TITLE
[blog] week 14

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]
:tulip:

File: content/posts/April-2022.md [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
+
+## Week 14 (April 5th - April 11th)
+
+- We have introduced two new build and test target aliases: `fedora-latest-stable` resolves to the latest stable Fedora
+  Linux release, while `fedora-branched` resolves to all branched releases (all Fedora Linux release, except `rawhide`).
+  ([packit#1546](https://github.com/packit/packit/pull/1546))
+- We have reverted the functionality of Packit that allowed you to have set **only** specific targets for Copr repositories.
+  This functionality was introduced a while ago and was found to be not very beneficial in cases of differently
+  configured Copr jobs building in parallel in the same Copr repository.
+  ([packit#1551](https://github.com/packit/packit/pull/1551))
```
Comment (lachmanfrantisek) [inline content/posts/April-2022.md:30]: @mfocko What about adding some "call-for-feedback"?

File: content/posts/April-2022.md [hunk 2]
```diff
@@ -1,20 +1,40 @@
...
+  ([packit#1542](https://github.com/packit/packit/pull/1542))
+
+## Week 14 (April 5th - April 11th)
+
+- We have introduced two new build and test target aliases: `fedora-latest-stable` resolves to the latest stable Fedora
+  Linux release, while `fedora-branched` resolves to all branched releases (all Fedora Linux release, except `rawhide`).
+  ([packit#1546](https://github.com/packit/packit/pull/1546))
+- We have reverted the functionality of Packit that allowed you to have set **only** specific targets for Copr repositories.
+  This functionality was introduced a while ago and was found to be not very beneficial in cases of differently
+  configured Copr jobs building in parallel in the same Copr repository.
```
Comment (TomasTomecek) [inline content/posts/April-2022.md:29]:
```suggestion
  configured Copr jobs building in parallel in the same Copr repository. This is an implementation detail in the end but Copr project chroots are now only added and never subtracted (every build has precisely specified targets based on the copr_build job definition).
```
Gotta say I had to open the PR to understand this one :D

**Review by TomasTomecek — APPROVED** [review]
just one suggestion which wasn't clear to me

File: content/posts/April-2022.md [hunk 3]
```diff
@@ -1,20 +1,40 @@
...
+- If you are using our stage instance, we make it listen only on `/packit-stg` comment commands so you can now differentiate
+  between the instances when commanding Packit via pull-request or issue comments.
+  For the production instance, you can continue using `/packit` prefix as you are used to.
+  ([packit-service#1432](https://github.com/packit/packit-service/pull/1432))
+- A new configuration option `downstream_branch_name` has been added, which is meant to be used in source-git projects
+  and allows users to customize the name of the branch in dist-git which corresponds to the current source-git branch.
+  ([packit#1555](https://github.com/packit/packit/pull/1555))
+- `revision-range` argument of the source-git `update-source-git` command is now optional. If not specified,
+  dist-git commits with no counterpart in source-git are synchronized.
+  ([packit#1547](https://github.com/packit/packit/pull/1547))
```
Comment (lachmanfrantisek) [inline content/posts/April-2022.md:40]: Do we want to cover source-git related changes as well? Or, make a dedicated part for `source-git`?

Comment (lbarcziova) [reply content/posts/April-2022.md:42]: I was thinking about this too, but we already cover the source-git related changes in our blogposts. Do we want to discuss this in the architecture?

**Pull request merged**